R/render-plot.R

```diff
@@ -37,10 +37,15 @@
 #'   is useful if you want to save an expression in a variable.
 #' @param func A function that generates a plot (deprecated; use \code{expr}
 #'   instead).
-#'
+#' @param replay If \code{TRUE} (the default), then when a plot is resized,
+#'   Shiny will \emph{replay} the plot drawing commands with
+#'   \code{\link[grDevices]{replayPlot}()} instead of re-executing \code{expr}.
+#'   This can result in faster plot redrawing, but there may be rare cases where
+#'   it is undesirable. If you encounter problems when resizing a plot, set this
+#'   to \code{FALSE}.
 #' @export
 renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
-                       env=parent.frame(), quoted=FALSE, func=NULL) {
+                       env=parent.frame(), quoted=FALSE, func=NULL, replay=TRUE) {
```

Comment:
Can we change the name of `replay` to something more to the effect of "do a full render on resize", and default it to `FALSE`?